components/script/dom/webglframebuffer.rs

```diff
@@ -21,7 +21,7 @@ use webrender_traits::{WebGLCommand, WebGLFramebufferBindingRequest, WebGLFrameb
 #[derive(JSTraceable, Clone, HeapSizeOf)]
 enum WebGLFramebufferAttachment {
     Renderbuffer(JS<WebGLRenderbuffer>),
-    Texture(JS<WebGLTexture>),
+    Texture(JS<WebGLTexture>, i32),
```

Comment:
A note saying this is the texture level would be nice. Alternatively:

`Texture { texture: JS<WebGLTexture>, level: i32 }`